coda/templates/mdstore/node.html

```diff
@@ -35,5 +35,9 @@ <h3 class="main">
             <th>Last Updated</th>
             <td> <i class="icon-calendar"></i> {{ node.last_checked }}</td>
         </tr>
+        <tr>
+	    <th>Status</th>
+	    <td> <span title="{{ node.status }}" class="disabled btn btn-primary btn-{% if node.status == "0" %}success{% else %}danger{% endif %}">{{ node.get_status_display }}</span></td>
+        </tr>

```

Comment:
I think we should swap the colors to where `status` 0 should be danger, and 1 should be success here and for the node_status.html.